### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by ritikarawat220**
✅ Add JavaScript classes instead of objects and arrays.
✅ Add objects and arrays inside classes.
✅ Create class methods to add and remove books
✅ Maintain all of the original functionality of the application
✅ Apply styles (colors, fonts, backgrounds, etc.),
✅ Pass all linters

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README, there are some sections of the README file that do not belong to the current project. Please update accordingly.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: class.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+window.addEventListener('DOMContentLoaded', () => {
+  Display.displayBooks();
+});
+const form = document.querySelector('.form');
+form.addEventListener('submit', (evt) => {
+  evt.preventDefault();
+  Display.addBook();
+  form.reset();
+});
```
Comment (Ibizugbe) [inline class.js:63]: currently, when I try to add a book, it doesn't work and I get an error message on the console saying " Cannot read properties of null (reading 'value') at Display.addBook" kindly fix this issue as it is required that even as we refactor our code to classes, we should retain the functionality of the application👍👍👍

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi Team👋, 

198009982-d99c2507-f20d-4c00-90a3-a54097598b36


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Pull request has a descriptive summary👍
- [x] Linters have been properly set up and they are passing👍
- [x] Javascript classes have been used instead of objects and arrays👍


## Required Changes ♻️

- [ ] You have done a great job on your readme but currently, there are still some issues giving it a less professional look😉
- Kindly add the right demo link or state that you don't the project hasn't been deployed yet😉 currently, the link is broken and this can lead to a bad user experience and give our project a less professional look🤗
- Some sections are not describing this project, kindly ensure your sections are filled and the content relates to this project😉 for example, the statement marked with yellow doesn't seem to relate to this project and looks a bit confusing😉
see the image below
bad_Readme1
- also consider the image below, I don't think `Awesome book` app  is a place to "display your web development project" that looks more like the portfolio project😉 kindly add a good description relating to this project🤗

_Check the comments under the review._

## Optional suggestions

- [ ] You can consider the use of array methods that don't mutate the array, e.g. concat and slice instead of push and splice as you have used.😉😉
- [ ] I suggest you use a unique `id` property for each book as it would make it easier to remove books with duplicate values.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

**Hi @Team,** 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**